_includes/partners-list.html

```diff
@@ -0,0 +1,30 @@
+<center class='each-partner'>
+  <div class="partner-post">
+    <img src="../img/cure-logo.png" class="partner-logo" >
```

Comment:
Each of these image elements needs an `alt` tag describing the image for accessibility. Also, they should have a width attribute to prevent the layout from jumping around as images load for users with slower connections.